### DISCUSSION TITLE
求问 怎么在taro框架中引入 vant3？

### DISCUSSION THREAD

**Discussion by mrchenxxx** [youzan/vant#10748] [Q&A] [answered]
### What problem does this feature solve?

求问 怎么在taro框架中引入 vant3？

### What does the proposed API look like?

求问 怎么在taro框架中引入 vant3？

**Comment by chenjiahan** [comment, marked as answer]
Vant 3 是基于 Vue 3 实现的，没有单独适配 taro 框架。
